SQL/migrate/V001__Initial_schema.sql

```diff
@@ -320,6 +320,7 @@ CREATE TABLE `ss13_characters` (
   `citizenship` text,
   `faction` text,
   `religion` text,
+  `accent` text,
```

Comment:
You need to create a new migration file which adds this column, instead of editing the initial migration.